---
layout: default
title: Méline WERY, CV
<a href="https://fulcrum.org" title="Fulcrum website">Fulcrum</a>
---
<div class="cv">
	<a href="/files/MWERY_2020.pdf" class="btn btn-default pull-right" title="Download CV as PDF">Download CV</a>
				<div itemscope itemtype="http://data-vocabulary.org/Person">
				<address>
				<span itemprop="affiliation"><strong>University of Angers, France</strong><br></span>
				<span itemprop="affiliation">SFR ICAT Platform<br></span>
				SFR 4208 <br>
				Bâtiment PBH-IRIS <br>
				4 Rue Larrey <br>
				CHU ANGERS <br>
				49933 Angers Cedex <br>
				<!-- <abbr title="phone">P:</abbr> (734) 763-4260<br> -->
				<span class="obfuscate">meline.wery@univ-angers.fr</span>
				</address>
				</div>
				
				<p>Research engineer in bio-informatics at SFR ICAT </p>
				
				<h2>Degrees</h2>
				<p><strong>PhD in Informatics, Rennes 1 University, Rennes, France, 2020</strong><br>
				<em>Identification of causal signature using omics data integration and network reasoning-based analysis</em></p>
				
				<p><strong>Master Degree in Bio-informatics and Genomics, University of Rennes 1, Rennes France, 2017</strong></p>

				<p><strong>Master Degree in Cellular and Molecular Biology, and Physiopathology, University of Angers, Angers France, 2014</strong></p>

				<h2>Professional Employment</h2>
				<p><strong>Research engineer in bio-informatics, SFR ICAT platform, University of Angers, 2021-present</strong></p>
				<ul>
				<li>Develop scripts and improve workflows for the analysis of biological data (RNAseq, single-cell ...) in python/R. </li>
				</ul>
				
				<p><strong>Engineer in bioinformatic, COMBI Team, LS2N University of Nantes, Nantes, oct-dec 2020</strong></p>
				<ul>
				<li>Creation of a integrative model of multiomic data for the IDEALG project.</li>
				</ul>
				
				<p><strong>PhD in Informatics, Dyliss team, INRIA/IRISA, Rennes &amp; SANOFI R&amp;R, Chilly-Mazarin, 2017-2020</strong>
				<em>Identification of causal signature using omics data integration and network reasoning-based analysis</em></p>
				<ul>
					<li>Creation of a integrative model of multiomic data using paired clinical trial datasets.</li>
					<li>Creation of a pipeline for the integration of multiomic data using a patient-centered approach.</li>
					<li>Identification of potential pathological signature through the interrogation to the integrative model</li>
				</ul>
				
				<p><strong>Internship in Bioinformatics, Dyliss team, INRIA/IRISA, Rennes France, 2017 (6 months)</strong>
				<em>Biological system signature at large scale</em></p>
				<ul>
					<li>Creation of a computational tool (FOCLASS) for the identification of signatures in a biological Boolean network.</li>
				</ul>
				
				<p><strong>Internship in Cellular and Molecular Biology, and Physiopathology, University of Angers, Angers France, 2014 (6 months)</strong>
				<em>Role of AGR2 in the anchorage-independent growth of cancer cells.</em></p>			
				<h2>Technology Applications and Expertise</h2>
					<dl class="dl-horizontal">
						<dt>Markup/Programming</dt>
						<dd>Python, R, Bash, ASP, LateX</dd>
						<dd>Git, GitHub, CVS</dd>
					</dl>	
					<dl class="dl-horizontal">
						<dt>Database</dt>
						<dd>RDF/SPARQL, SQL</dd>
					</dl>
					<dl class="dl-horizontal">
						<dt>Applications</dt>
						<dd>GINSIM, Array Studio</dd>
						<dd>Inkscape, Atom, Visual Studio Code, Microsoft Office</dd>
					</dl>
					<dl class="dl-horizontal">
						<dt>OS/Environments</dt>
						<dd>Linux, Unix, Ubuntu, Mac, Windows</dd>
					</dl>
					
</div>
